meson.build

```diff
@@ -367,13 +367,24 @@ if get_option('enable-app')
   endif
 endif
 
-if get_option('enable-nnstreamer-tensor-filter')
-  if get_option('platform') == 'android'
-    warning('android nnstreamer-filter is not yet supported, building nnstreamer-filter skipped')
-  else
-    nnstreamer_dep = dependency('nnstreamer', required: true)
-    subdir('nnstreamer/tensor_filter')
-  endif
+if get_option('enable-nnstreamer-tensor-filter').enabled()
+	if get_option('platform') == 'android'
+		warning('android nnstreamer-filter is not yet supported, building nnstreamer-filter skipped')
+	else
+    nnstreamer_filter_dep = dependency('nnstreamer')
+		subdir('nnstreamer/tensor_filter')
+		extra_defines += '-DSUPPORT_NNSREAMER_PLUGIN_FILTER=1'
+	endif
+endif
+
+if get_option('enable-nnstreamer-tensor-trainer').enabled()
+	if get_option('platform') == 'android'
+		warning('android nnstreamer-trainer is not yet supported, building nnstreamer-trainer skipped')
+	else
+    nnstreamer_trainer_dep = dependency('nnstreamer')
+		subdir('nnstreamer/tensor_trainer')
```

Comment:
you might need add the ```meson.build``` in ```nnstreamer dir``` and just ```subdir('nnstreamer')``` here.

Reply:
okay, thank you.

Reply:
done ([42e6353](https://github.com/nnstreamer/nntrainer/pull/2059/commits/42e63538cf5f5da059f1b23ced491724ef751b87))
When the tensor_trainer sub-plugin is stabilized, we will merge it with the tensor_filter sub-plugin package.